src/frontend/ui/commun/IndicateurGraphique/IndicateurGraphique.module.css

```
@@ -4,8 +4,11 @@
 
 .titraille {
   margin-bottom: .5rem;
-}
-
-.titraille abbr {
   margin-right: .5rem;
 }
+
+.mise-à-jour-source {
```

```suggestion
.mise-a-jour-source {
```
Dans le navigateur, il le traduit pas un `_`

---

il faudrait le mettre dans le CONTRIBUTING qu'on ne met pas d'accents dans le css alors